Doc/library/json.rst

```diff
@@ -678,40 +678,42 @@ when serializing instances of "exotic" numerical types such as
 
 
 .. _json-commandline:
-.. program:: json.tool
+.. program:: json
 
-Command Line Interface
+Command-line interface
 ----------------------
 
-.. module:: json.tool
-    :synopsis: A command line to validate and pretty-print JSON.
-
 **Source code:** :source:`Lib/json/tool.py`
```

Comment:
Even though it can be accessed as `python -m json` now, this is still the `json.tool` submodule documentation (the source reference doesn't really make sense otherwise).

```suggestion
.. module:: json.tool
    :synopsis: A command line interface to validate and pretty-print JSON.

**Source code:** :source:`Lib/json/tool.py`
```

Reply:
Same thing with hyphenated "command-line interface":

```suggestion
.. module:: json.tool
    :synopsis: A command-line interface to validate and pretty-print JSON.

**Source code:** :source:`Lib/json/tool.py`
```